active/003-seq-feature.md

```diff
@@ -0,0 +1,180 @@
+---
+Feature name: sequence-positional-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt"]
```

Comment:
Contributors should be those who participated in discussion on GitHub excluding the authors who are already listed above.